MatrixGame/src/MatrixDebugInfo.cpp

```diff
@@ -21,8 +21,8 @@ CMatrixDebugInfo::CMatrixDebugInfo()
 : m_Font{nullptr}
 , m_Pos{DI_KEY_X, DI_KEY_Y}
 {
-    if (D3D_OK != D3DXCreateFontW(g_D3DD, 10, 0, FW_NORMAL, 1, FALSE, DEFAULT_CHARSET, OUT_DEFAULT_PRECIS,
-                                  DEFAULT_QUALITY, DEFAULT_PITCH, L"MS Sans Serif", &m_Font))
+    if (D3D_OK != D3DXCreateFontW(g_D3DD, 30, 0, FW_NORMAL, 1, FALSE, DEFAULT_CHARSET, OUT_DEFAULT_PRECIS,
```

Comment:
TBH i'm not sure if such a big text is necessary (on my FHD screen it looks HUGE), but i don't mind.  
just please make sure `DI_KEY_W` is big enough for the keys to not overlap with values at least for the cheats (e.g. try `KEEPALIVE` cheat)

Reply:
I have FHD too, but it isn't big at all, like ~2-3% of window height(in exe) per row.
Anyway its matter of tastes
other point is valid